docs/book/quick_start.md

```diff
@@ -25,7 +25,7 @@ curl -L -O https://github.com/kubernetes-sigs/kubebuilder/releases/download/v${v
 
 # extract the archive
 tar -zxvf kubebuilder_${version}_darwin_${arch}.tar.gz
-sudo mv kubebuilder_${version}_darwin_${arch} /usr/local/kubebuilder
+mv kubebuilder_${version}_darwin_${arch} kubebuilder && sudo mv kubebuilder /usr/local/
```

Comment:
I don't understand the need for changing this... could you explain?x

Reply:
Oh，sorry. I just follow the guide to install kubebuilder. But the package extract as `/usr/local/kubebuilder/kubebuilder_1.0.6_linux_amd64/bin`. Maybe there are something wrong in my env.
I clean the `/usr/local/kubebuilder/` dir and try again, it can be installed successfully. I'll abandon this PR. Thanks for you comments.

Reply:
It's OK, and now I understand why you want to change this. But I'm afraid this introduces no differences.

I think maybe it's because you had the directory `/usr/local/kubebuilder` before, and you were running `sudo mv kubebuilder_${version}_darwin_${arch} /usr/local/kubebuilder/` instead of `sudo mv kubebuilder_${version}_darwin_${arch} /usr/local/kubebuilder`.

Reply:
Yes, maybe. Thanks anyway~